app/src/pages/metamine/visualizationNU/UmapPlot.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+    <div class="md-gutter adjust-padding wrapper-box">
+      <div class="main-content" style="margin: 3rem; display: flex">
+        <div class="umap-chart" style="width: 50%">
+          <Umap />
+          <div style="display:flex; justify-content: space-between; margin-top: 40px;">
+            <button @click="toggleDialogBoxKnn()" class="nuplot-button">Find Nearest Neighbors</button>
+          <dialog-box :active="dialogBoxActiveKnn">
+            <template v-slot:content>
+              <NeighborPanel />
+            </template>
+            <template v-slot:actions>
+              <md-button @click.native.prevent="toggleDialogBoxKnn">Close</md-button>
+            </template>
+          </dialog-box>
+          <button @click="toggleDialogBoxSaveData()" class="nuplot-button button-primary">Save Data</button>
+          <dialog-box :active="dialogBoxActiveSaveData">
```

Comment:
Similar comment to other dialog boxes; recommend adding `:disableClose="true"` to these since you're using custom toggling instead of store